fastlane-config/ios_config.rb

```
@@ -1,37 +1,44 @@
+      issuer_id: ProjectConfig::IOS_SHARED[:app_store_connect][:issuer_id],
+      key_filepath: ProjectConfig::IOS_SHARED[:app_store_connect][:key_filepath],
+      version_number: ProjectConfig::IOS[:version_number],
+      metadata_path: ProjectConfig::IOS[:metadata_path],
+      app_rating_config_path: ProjectConfig::IOS[:age_rating_config_path],
       screenshots_ios_path: "./fastlane/screenshots_ios",
       screenshots_macos_path: "./fastlane/screenshots_macos",
+      primary_locale: ProjectConfig::IOS[:primary_locale],
+
+      # TestFlight Configuration (from IOS_SHARED)
+      TESTFLIGHT_CONFIG = ProjectConfig::IOS_SHARED[:testflight]
+
```

RuboCop (1.84.2)

[fatal] unexpected token tNL
(Using Ruby 3.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)